packages/dev-middleware/src/createDevMiddleware.js

```diff
@@ -61,11 +62,11 @@ type Options = $ReadOnly<{
   unstable_experiments?: ExperimentsConfig,
 
   /**
-   * An interface for using a modified inspector proxy implementation.
+   * Create a middleware for handling unsupported CDP events, per device.
```

Comment:
> Pedantic nit: It looks like we actually give the middleware precedence over passing messages to/from the app, rather than just passing "unsupported" events to it.

Yes, unfortunately, we need to be able to influence "semi-supported" events too. It's mostly for vscode debugging, as some events are responded but not fully understood by `vscode.js-debug`. E.g. https://github.com/microsoft/vscode-js-debug/issues/1583

There are more cases, or there could be more popping up. Having the ability to mutate or "replace"/workaround incompatible requests/responses would help us maintain the vscode debugging method.

Reply:
> Per page, right?

✅ I updated the docs